lumi/testbench/test_lumi.py

```diff
@@ -4,16 +4,17 @@
 # Copyright (C) 2023 Zero ASIC
 # This code is licensed under Apache License 2.0 (see LICENSE for details)
 
+import random
 import numpy as np
 from pathlib import Path
 from argparse import ArgumentParser
-from switchboard import SbDut, UmiTxRx, delete_queue, verilator_run, binary_run
+from switchboard import SbDut, UmiTxRx, delete_queue
 
 THIS_DIR = Path(__file__).resolve().parent
 
 
-def build_testbench(topo="2d"):
-    dut = SbDut('testbench')
+def build_testbench(topo="2d", trace=False):
+    dut = SbDut('testbench', trace=True, trace_type='fst', default_main=True)
```

Comment:
Minor comment, Can you pass the argument trace from build_testbench to the SbDut?

Reply:
This was done because of what seems to be a verilator bug. Without trace the umi tx block does not see the transactions so this is a temp wa.